manifests/image-references

```diff
@@ -30,3 +30,7 @@ spec:
     from:
       kind: DockerImage
       name: registry.ci.openshift.org/openshift:gcp-cluster-api-controllers
+  - name: ibmcloud-cluster-api-controllers
+    from:
+      kind: DockerImage
+      name: registry.ci.openshift.org/openshift:ibmcloud-cluster-api-controllers
```

Comment:
Is this in payload already?

Reply:
Its almost there, Once [this](https://github.com/openshift/ocp-build-data/pull/2072) PR is merged I think we would be seeing the image in payload